google/datalab/utils/commands/_utils.py

```diff
@@ -328,6 +329,8 @@ def parse_config(config, env, as_dict=True):
     config = {}
   elif stripped[0] == '{':
     config = json.loads(config)
+  elif six.PY3:
+    config = yaml.load(config, Loader=yaml.FullLoader)
```

Comment:
What are these loader changes addressing? This seems unrelated to Python 3.7 compat to me at first glance.

Reply:
I am sorry that I cannot remember the reason clearly.
But I searched on google and found an issue concerning pyyaml version.

* https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation
* https://github.com/yaml/pyyaml/issues/292

I remember that I built the datalab image with py3 and got into some troubles with pyyaml before.

As the pyyaml version show as below,
https://github.com/googledatalab/pydatalab/blob/8c2df84a1f3bc4db6eb3b4d139676826f8c2e222/setup.py#L108

I guess that pyyaml will be installed as different version in py2 and py3.

As you could see in the py35 coverage test, there were warnings concerning YAMLLoadWarning.
```
/home/travis/build/googledatalab/pydatalab/tests/pipeline/pipeline_tests.py:305: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  dag_dict = yaml.load(PipelineTest._test_pipeline_yaml_spec)
```
```
/home/travis/build/googledatalab/pydatalab/tests/pipeline/pipeline_tests.py:265: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  dag_dict = yaml.load(PipelineTest._test_pipeline_yaml_spec)
```
```
/home/travis/build/googledatalab/pydatalab/tests/pipeline/pipeline_tests.py:280: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  dag_dict = yaml.load(PipelineTest._test_pipeline_yaml_spec)
```

Maybe there will be warnings or errors in py37, too.
I'm sorry that maybe it is better to open a new PR concerning the YAMLLoadWarning thing.
And it is better add some py37 coverage tests.